spinkit.css

```diff
@@ -572,4 +573,100 @@
   } 100% {
     transform: rotate(-360deg); 
   }
+}
+
+/*  Hexagon
+
+      <div class="sk-hexagon">
+        <div class="sk-hex-parent">
+          <div class="sk-hex-child"></div>
+          <div class="sk-hex-child"></div>
+          <div class="sk-hex-child"></div>
+        </div>
+      </div>
+ */
+.sk-hexagon {
+  height: var(--sk-size);
+  overflow: hidden;
+  position: relative;
+  width: var(--sk-size);
+}
+
+.sk-hex-parent {
+  left: calc(27 * var(--sk-hex-scale));
+  position: relative;
+  top: calc(16 * var(--sk-hex-scale));
+}
+
+.sk-hex-child {
+  animation: sk-hexagonAnimation 2s infinite cubic-bezier(.19, 1.6, .4, 1);
+  background-color: var(--sk-color);
+  display: block;
+  height: calc(26 * var(--sk-hex-scale));
+  opacity: 0;
+  position: absolute;
+  transform: scale(0);
+  transform-origin: center center;
+  width: calc(48 * var(--sk-hex-scale));
+}
+
+.sk-hex-child::before,
+.sk-hex-child::after {
+  border-left: calc(24 * var(--sk-hex-scale)) solid transparent;
+  border-right: calc(24 * var(--sk-hex-scale)) solid transparent;
+  content: "";
+  height: 0;
+  position: absolute;
+  width: 0;
+}
+
+.sk-hex-child::before {
+  border-bottom: calc(13 * var(--sk-hex-scale)) solid var(--sk-color);
+  top: calc(-13 * var(--sk-hex-scale));
+}
+
+.sk-hex-child::after {
+  border-top: calc(13 * var(--sk-hex-scale)) solid var(--sk-color);
+  bottom: calc(-13 * var(--sk-hex-scale));
+}
+
+.sk-hex-child:nth-child(2),
+.sk-hex-child:nth-child(3) {
+  top: calc(45 * var(--sk-hex-scale));
```

Comment:
There's a lot of `calc` usage in general here to position the three hexagons—could we just use %? They're just being centered horizontally or attached to a corner, right?